### ISSUE TITLE
Inpainting failing for a particular texture

### ISSUE THREAD

**Issue by PetrVevoda** [SHI-Labs/FcF-Inpainting#13]
Dear authors,

I tried your Colab demo and ran into a problem. When I tried to inpaint the following image:
![texture_dark](https://user-images.githubusercontent.com/7972359/207414580-0099c98f-3f27-4571-a409-e99ed88af9fb.png)
with this mask:
![texture_dark_mask](https://user-images.githubusercontent.com/7972359/207414587-6d787b60-09e6-42bb-a080-11008a9b0668.png)
I got quite a good result:
![texture_dark_inpainted](https://user-images.githubusercontent.com/7972359/207414585-b7c76d93-0c2f-4436-a835-8b5554d1662b.png)

However, if I just made the input image brighter (multiplied pixel values by 2):
![texture_bright](https://user-images.githubusercontent.com/7972359/207414678-1b0d41d9-fa74-4190-a76e-5e70c6fda804.png)
the inpainting (with the same mask) failed:
![texture_bright_inpainted](https://user-images.githubusercontent.com/7972359/207414681-0c483d57-dc4d-46bf-af4b-1690150de472.png)

I thought it could be some overflow in the image domain, but when I tried to crop the input image a bit (about 10 % on the top and right side):
![texture_bright_crop](https://user-images.githubusercontent.com/7972359/207414701-efe51546-1de0-4c12-a910-73997cd26fcb.png)
the inpainting worked again:
![texture_bright_crop_inpaint](https://user-images.githubusercontent.com/7972359/207414703-85b43a16-409d-4108-b5c1-422b938de541.png)

Could be some overflow in the frequency domain or am I just doing something wrong? Thank you for your response.

**Comment by praeclarumjj3** [comment]
Hi @PetrVevoda, thanks for your interest in our work. I did encounter a similar issue while testing myself for some kinds of masks for some reason. I have not had the time to look into the case yet. However, it was solved for me by changing the seed AFAIR. Did you try changing the seed?

The most likely reason seems to be some explosion in the pixel values similar to what you suggested.

**Comment by PetrVevoda** [comment]
Yes, I tried several seeds but I only got slightly different noise patterns.

**Comment by praeclarumjj3** [comment]
Right, I think it's a mask and image-specific issue. I will look into this when I get the chance. Can you share the image and mask with me?

**Comment by PetrVevoda** [comment]
Sure, here they are:
![texture_bright](https://user-images.githubusercontent.com/7972359/207845159-b65d899d-2134-4108-81d7-76142d837f4e.png)
![texture_bright_mask](https://user-images.githubusercontent.com/7972359/207845162-537b4318-f32d-42a3-af14-6462b9dc3c57.png)
